lms/static/sass/base/_base.scss

```diff
@@ -183,12 +183,9 @@ span.edx {
 }
 
 .loading-animation {
```

Comment:
are there any other places outside of discussions that this is being used / referenced? If so we might want to proof / check those pages as well.